Rock/Reporting/EntityHelper.cs

```diff
@@ -50,7 +50,7 @@ public static List<EntityField> GetEntityFields( Type entityType, bool includeOn
 
             if ( HttpContext.Current != null )
             {
-                entityFields = HttpContext.Current.Items[string.Format( "EntityHelper:GetEntityFields:{0}", entityType.FullName )] as List<EntityField>;
+                entityFields = HttpContext.Current.Items[string.Format( "EntityHelper:GetEntityFields:{0}_{1}_{2}", entityType.FullName, includeOnlyReportingFields, limitToFilterableFields )] as List<EntityField>;
```

Comment:
@nairdo Yes, thanks for catching this.  I've committed another change to address this.